### PR TITLE
Updated and slight reword around the consultation for starting a new node project

### PR DESCRIPTION
**Pull request by Nooshu**
Removed the reference to Deputy Director Technology Operations.

### CODE REVIEW

File: source/manuals/programming-languages/nodejs/index.html.md.erb [hunk 1]
```diff
@@ -6,8 +6,8 @@ title: Using Node.js at GDS
 
 ## Important Note
 
-No new projects at GDS should use Node.js without prior consultation with the 
-Deputy Director Technology Operations. The [programming language recommendations](../../../standards/programming-languages.html#javascript) 
+New projects at GDS should never use Node.js without prior consultation with the 
+GDS Way Forum. Topics like this are discussed at our monthly meetings. The [programming language recommendations](../../../standards/programming-languages.html#javascript)
```
Comment (nickcolley) [inline source/manuals/programming-languages/nodejs/index.html.md.erb]: How do you consult the GDS Way Forum?

Comment (Nooshu) [reply source/manuals/programming-languages/nodejs/index.html.md.erb]: Good question... @pgchamberlin / @rboulton how do we consult the forum :)

Comment (pgchamberlin) [reply source/manuals/programming-languages/nodejs/index.html.md.erb]: Email to `programme-technical-leaders@digital.cabinet-office.gov.uk` explaining the need and linking to whatever else might be relevant.

Comment (Nooshu) [reply source/manuals/programming-languages/nodejs/index.html.md.erb]: Updated

**Pull request merged**